server/src/main/java/org/candlepin/pinsetter/core/PinsetterKernel.java

```diff
@@ -67,6 +67,7 @@
 
     public static final String CRON_GROUP = "cron group";
     public static final String SINGLE_JOB_GROUP = "async group";
+    public static final String[] DELETED_JOBS = new String[] { "StatisticHistoryTask" };
```

Comment:
@alikins I did consider it, but thought that it was overkill.
* I figured that deleting a job to this extent is a code change, so it should not be an additional challenge to add a string to this array.
* using a config I thought would be an overkill too, given we would / should only change that config only every time we delete a job class.
* even if we add a config, we will hard code the defaults in ConfigProperties, in which case I figured its better to hard code with the code that uses it.

@alikins further thoughts?